src/config.js

```diff
@@ -0,0 +1,16 @@
+// It would be cleaner for this to live inside './queries', but
+// other parts of the code assume that all exports from
+// './queries' are query functions.
+let config = {
+  testIdAttribute: 'data-testid',
+}
+
+export function configure(configDelta) {
+  config = {
+    ...config,
+    ...configDelta,
+  }
+}
+export function getTestIdAttribute() {
+  return config.testIdAttribute
```

Comment:
This looks good to me. My only nit is that I don't think there needs to be a special method to grab just this attribute; export the whole config object or a getConfig function instead.

Reply:
Configure could also take an update function:

```js
configure(currentConfig => { return {...newConfig} })
```

This would be nice if you want to read and extend subobjects or arrays (like the current set of queries) without importing the default value into the host file.